run_all_tools.sh

```diff
@@ -97,62 +103,110 @@ function run_cloudrail {
 }
 
 function run_all {
-  run_checkov
-  run_cloudrail
-  run_kics
-  run_snyk
-  run_terrascan
-  run_tfsec
+  echo Now running Checkov on all cases
+  run_checkov $1
+  echo Now running Cloudrail on all cases
+  run_cloudrail $1
+  echo Now running KICS on all cases
+  run_kics $1
+  echo Now running Snyk on all cases
+  run_snyk $1
+  echo Now running Terrascan on all cases
+  run_terrascan $1
+  echo Now running TFsec on all cases
+  run_tfsec $1
 }
 
-# Set up AWS access for plan
-if [ -z "$AWS_ACCESS_KEY_ID" ]; then
-  echo "To run this script, you'll need AWS credentials (for use with terraform plan)."
-  exit 1
-fi
-export AWS_REGION=us-west-1
+if [[ $1 == *"all" ]] || [[ $1 == *"cloudrail" ]]; then
```

Comment:
It's actually not cloudrail that needs the AWS_ACCESS_KEY_ID, it's the plan generation.
Cloudrail needs the plan, but I'm anticipating other tools will switch to using the plan more soon too (like Checkov recently started doing).

Reply:
I understand your point, some other tools require the terraform plan output too. What if I don't have an AWS account setup at the moment and I want to compare different tools that don't require terraform plan, could you suggest another way we could fit these two use cases?

Reply:
That's a great question and is related to #21 . Many of the tools listed in this repository (and others) also support looking at live cloud environments as part of an infrastructure-as-code security practice. We will likely need to create an AWS account everyone can read from in order to support this use case. I was hoping to get feedback on #21 on this matter.